src/tests/sys_tests.rs

```diff
@@ -90,5 +90,14 @@ pub mod test_sys {
         assert_eq!(child_cage.cwd.read(),cage.cwd.read())
     }
 
+    pub fn ut_lind_exit() {
+        // Since exit function is heavily used and tested in other syscalls and their tests 
+        // We only perform preliminary checks for checking the sanity of this syscall
+        lindrustinit(0);
+        let cage = interface::cagetable_getref(1);
+        // Call the exit call
+        assert_eq(cage.exit_syscall(EXIT_SUCCESS),EXIT_SUCCESS); 
+        lindrustfinalize(); 
```

Comment:
Just curious, since we are deleting a couple of references from the cage like "stdout stream", "associated file descriptors", it is possible to validate them through a test if they are getting deleted or not successfully in the backend?

Reply:
I think once we exit a cage - we completely lose all locks and fds that the process held - and hence can't double check or read them later once the cage object has been removed from the cage table